### PR TITLE
Fixed Issue #1193: Added space below Lock Now button 

### LINKED ISSUE #1193: "Lock now" button is too close to the screen margin
## Preconditions:
- Set-up Passcode and Touch ID. 
## Steps to reproduce
1. Connect to Lockwise
2. Go to Settings
### Expected behavior
- "Lock now" should be a bit scrolled up into view on iPhone 8 and SE screen size. 
### Actual behavior
- "Lock now" button is too close to the screen margin.
### Device & build information
* Device: iPhone 8, iPhone SE
* Build version: master d14e787c20
### Notes
- If Passcode/Touch ID is not set-up this issue is not reproducible.
Attachments: 
- Screen Shot 2020-02-12 at 4 55 50 PM

### PR DESCRIPTION
**Pull request by KrisJackson**
Fixes #1193 

Added space below Lock Now button in settings.

## Testing and Review Notes

Go to Settings page. Scroll to bottom on TableView. There should be a space of 18px below the last cell.

## Screenshots or Videos

Simulator Screen Shot - iPhone 8 - 2020-05-18 at 23 37 49

### CODE REVIEW

File: lockbox-ios/View/ItemDetailView.swift [hunk 1]
```diff
@@ -194,7 +194,8 @@ extension ItemDetailView: UIGestureRecognizerDelegate {
                 }
                 
                 if let revealObserver = cellConfiguration.revealPasswordObserver {
-                    cell.textValue.font = UIFont(name: "Menlo-Regular", size: 16)
+                    // Font is not consistent with app. Uncomment this line out if this was intentional 
+//                    cell.textValue.font = UIFont(name: "Menlo-Regular", size: 16)
```
Comment (eliserichards) [inline lockbox-ios/View/ItemDetailView.swift:198]: I think these lines can just be deleted 👍

Comment (KrisJackson) [reply lockbox-ios/View/ItemDetailView.swift]: Done!

Comment (KrisJackson) [reply lockbox-ios/View/ItemDetailView.swift]: @eliserichards For some reason I keep getting "build error" for the uispecs test in GitHub. Is that error on my side?

Comment (kaylagalway) [reply lockbox-ios/View/ItemDetailView.swift]: @KrisJackson  no it isn't - figuring it out now. It's an error with our python script that creates fake logins for the main list, I believe

**Pull request closed without merging**